components/automate-ui/src/app/entities/rules/rule.reducer.ts

```diff
@@ -76,7 +78,8 @@ export const RuleEntityInitialState: RuleEntityState = ruleEntityAdapter.getInit
   createStatus: EntityStatus.notLoaded,
   createError: null,
   deleteStatus: EntityStatus.notLoaded,
-  updateStatus: EntityStatus.notLoaded
+  updateStatus: EntityStatus.notLoaded,
+  projectRulesStatus: <ProjectStatus>'PROJECT_RULES_STATUS_UNSET'
```

Comment:
❓ Is this needed? `RuleEntityState` has that type, too, so I'd suppose this would be sufficient to guarantee you don't just set anything here....?

Reply:
It actually is needed; that surprised me, too. There's a compiler error without it.

Reply:
Ah, thanks for noticing that. It actually is needed there because the system does not know this is a `RuleEntityState`--it is just an object we are passing to `ruleEntityAdapter.getInitialState` and that has an untyped argument.
That said, there is a better way to write this so it is not needed--see new commit for that.